duktape.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: duktape
+  version: "2.7.0"
+  epoch: 0
+  description: "Embeddable Javascript engine with a focus on portability and compact footprint"
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 90f8d2fa8b5567c6899830ddef2c03f3c27960b11aca222fa17aa7ac613c2890
+      uri: https://github.com/svaarala/duktape/releases/download/v${{package.version}}/duktape-${{package.version}}.tar.xz
```

Comment:
Is this manually created download artifact with generated things or stuff not in the git repo, or a straight up GitHub tag export?

If it just git tag export, we prefer to build from a git-checkout.